source/auth/auth.rst

```diff
@@ -1272,30 +1251,49 @@ mechanism
 mechanism_properties
     PROVIDER_NAME
         Drivers MUST allow the user to specify a name for using a service
-        to obtain credentials that is one of "aws", "azure", or "gcp".
+        to obtain credentials that is one of ["aws"].
     REQUEST_TOKEN_CALLBACK
         Drivers MUST allow the user to specify a callback of the form
-        "onOIDCRequestToken" (defined below), if the driver supports
-        providing objects as mechanism property values.
+        "onRequest" (defined below), if the driver supports
+        providing objects as mechanism property values.  Otherwise the driver MUST allow it as a MongoClientOption.
     REFRESH_TOKEN_CALLBACK
         Drivers MUST allow the user to specify a callback of the form
-        "onOIDCRefreshToken" (defined below), if the driver supports
-        providing objects as mechanism property values.
-
-Drivers MUST skip client step 1 when using a service to obtain credentials
-or when the server step 1 response value is cached.  When skipping step 1,
-drivers will use ``saslStart`` and a payload with the ``jwt`` value.
+        "onRefresh" (defined below), if the driver supports
+        providing objects as mechanism property values.  Otherwise the driver MUST allow it as a MongoClientOption.
+    ALLOWED_HOSTS
+        The list of allowed hostnames or ip-addresses (ignoring ports) for
+        MongoDB connections. The hostnames may include a leading "*." wildcard, which allows for matching subdomains. ALLOWED_HOSTS is a
```

Comment:
At least Github's RST processing seems to process the `*` as an italics marker

Also, it might be good to clarify:

```suggestion
        MongoDB connections. The hostnames may include a leading "*." wildcard, which allows for matching (potentially  nested) subdomains. ALLOWED_HOSTS is a
```